release-notes.adoc

```diff
@@ -48,8 +45,7 @@ Portal client
 * 
 
 [[mobile-client]]
-Mobile client
-^^^^^^^^^^^^^
+=== Mobile client
```

Comment:
@pmdarrow, the edits are there, see https://github.com/feedhenry-raincatcher/raincatcher-documentation/blob/67c87d1c31537206b8d928d7b6482b59a0a7de98/release-notes.adoc
(think it might be an issue with github rendering, I've started a repo to see what does and doesn't work https://github.com/finp/asciidocTest)